src/Data/Textile/Inputs.elm

```diff
@@ -319,15 +320,15 @@ weavingOptionsToString _ _ =
 makingOptionsToString : Inputs -> String
 makingOptionsToString { product, makingWaste, airTransportRatio, disabledFading } =
     [ makingWaste
-        |> Maybe.map (Format.ratioToPercentString >> (\s -> s ++ " de perte"))
+        |> Maybe.map (Split.toPercentString >> (\s -> s ++ "\u{202F}% de perte"))
     , airTransportRatio
         |> Maybe.andThen
             (\ratio ->
-                if Unit.ratioToFloat ratio == 0 then
+                if Split.asPercent ratio == 0 then
                     Nothing
 
                 else
-                    Just (Format.ratioToPercentString ratio ++ " de transport aérien")
+                    Just (Split.toPercentString ratio ++ " de transport aérien")
```

Comment:
Nit: maybe rename `ratio` to `split` in such circumstances to ease readability a bit

Reply:
Not sure about that: the record field name (that corresponds to the API, and the json data) is still `airTransportRatio`.

Reply:
I won't die on this hill, please land this patch 😄